README.md

```diff
@@ -1,20 +1,38 @@
-# retry-js
+# @codewell/retry
+
 Recursively retries a function if it fails
-  # @codewell/retry-js
 
-  ## Installation
-  ```
-  npm install @codewell/retry-js
-  ```
+## Installation
+
+```
+npm install @codewell/retry
+```
+
+## Basic Usage
+
+```JavaScript
+import retry from '@codewell/retry';
+
+const foo = (name) => {
```

Comment:
might be missing something, but have an example function that actual exercises the main functionality and explains what it does just by looking at the example and the result

Reply:
Good idea!